uniffi_bindgen/src/bindings/python/templates/macros.py

```diff
@@ -53,6 +53,21 @@
     {%- endfor %}
 {%- endmacro %}
 
+{#-
+// Field lists as used in Python declarations of Records and Enums.
+// Note the var_name and type_name filters.
```

Comment:
Maybe remove the "and type_name" part.  I don't think you end up using that filter since it's Python.

Reply:
classic copy-pasta error.